remove optimize c7 material from 3.14 on (#5377)
camunda/camunda-docs: docs/guides/improve-processes-with-optimize.md

### Users

Within the **Users** tab, review the users with access to your collection.

> Select **Add** to search for a user to add, of which may be assigned as a viewer, editor, or manager.

Reviewer: ```suggestion
Select **Add** to search for a user to add, who can be assigned as a viewer, editor, or manager.
```

Reviewer: Suggestion for rewording
